src/components/users/users-directive.tpl.html

```diff
@@ -2,35 +2,39 @@
   <table class="table table-striped table-bordered table-fixed b-t" refresh-on="OrganizationChanged UserChanged" refresh-action="vm.get(vm.currentOptions)" refresh-throttle="10000">
     <thead>
       <tr>
-        <th>Email Address</th>
-        <th class="action">Actions</th>
+        <th>{{'Email Address' | translate}}</th>
+        <th>{{'Full Name' | translate}}</th>
```

Comment:
Fullname shouldn't be here :)

Reply:
But in china,the email address can't be identified people. There are QQ and custom email name. So I add Full Name column in this.

Reply:
@Varorbc looking at this again, We'd feel more comfortable if this and a few other changes were reverted (we can talk about them more in a new issue).